src/angular-cesium/angular-cesium.module.ts

```diff
@@ -95,6 +96,7 @@ import { ConfigurationService } from './cesium-enhancements/ConfigurationService
     AcPolylineVolumeDescComponent,
     AcWallDescComponent,
     AcRectangleDescComponent,
+    AcContextMenuWrapperComponent,
```

Comment:
maybe should be in another module somthing like `AngularCesiumWigets` 
what do you think? ... another task

Reply:
yeah we should - another task